### PR TITLE
ci: Add issue and PR assigner

### PR DESCRIPTION
**Pull request by nordicjm**
Adds a file with codeowners

    Adds actions that will automatically assign users to reviews
    and issues

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+boot/boot_serial @nordicjm @de-nordic
+boot/bootutil @davidvincze
+boot/cypress @romanjoe
+boot/espressif @almir-okato
+boot/mbed @AGlass0fMilk
+boot/mynewt @kasjer
+boot/muttx @michallenc
```
Comment (d3zd3z) [inline CODEOWNERS:7]: Presumably, this should be `boot/nuttx` with an 'n'.

Comment (nordicjm) [reply CODEOWNERS]: Fixed

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+jobs:
+    auto-assign:
+        runs-on: ubuntu-latest
+        permissions:
+            issues: write
+        steps:
+          - uses: actions/checkout@v1
+          - uses: ohnogumi/auto-reviewers@v0.0.5
+            with:
+                token: "${{ secrets.USER_TOKEN }}"
```
[the remark this answers is not in the document]
Comment (nordicjm) [reply .github/workflows/pr.yml]: Removed the quotes but still failing, did you add this under mcuboot -> settings -> secrets and variables -> actions -> repository secrets?

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,9 @@
+boot/boot_serial @nordicjm @de-nordic
+boot/bootutil @davidvincze
+boot/cypress @romanjoe
+boot/espressif @almir-okato
+boot/mynewt @kasjer
+boot/nuttx @michallenc
```
[the remark this answers is not in the document]
Comment (nordicjm) [reply CODEOWNERS:6]: According to https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization#permissions-for-each-role only read/triage permission should be needed. David has sent an invite to them in the comment above

File: .github/workflows/pr.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (nordicjm) [reply .github/workflows/pr.yml]: Hmm I've changed to a test action that I have tested and is known working in my own repo and I'm getting the same error that the parameter is empty

**Pull request closed without merging**